haproxy.cfg

```diff
@@ -28,21 +28,20 @@ defaults
     load-server-state-from-file global
 
     # Use provided example error pages
-    errorfile 400 /usr/local/etc/haproxy/errors/400.http
-    errorfile 403 /usr/local/etc/haproxy/errors/403.http
-    errorfile 408 /usr/local/etc/haproxy/errors/408.http
-    errorfile 500 /usr/local/etc/haproxy/errors/500.http
-    errorfile 502 /usr/local/etc/haproxy/errors/502.http
-    errorfile 503 /usr/local/etc/haproxy/errors/503.http
-    errorfile 504 /usr/local/etc/haproxy/errors/504.http
+    # errorfile 400 /usr/local/etc/haproxy/errors/400.http
+    # errorfile 403 /usr/local/etc/haproxy/errors/403.http
+    # errorfile 408 /usr/local/etc/haproxy/errors/408.http
+    # errorfile 500 /usr/local/etc/haproxy/errors/500.http
+    # errorfile 502 /usr/local/etc/haproxy/errors/502.http
+    # errorfile 503 /usr/local/etc/haproxy/errors/503.http
+    # errorfile 504 /usr/local/etc/haproxy/errors/504.http
 
 backend dockerbackend
     server dockersocket /var/run/docker.sock
 
 frontend dockerfrontend
     bind :2375
-    http-request deny unless METH_GET || { env(POST) -m bool }
-    http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/containers/[a-zA-Z0-9_.-]+/((stop)|(restart)|(kill)) } { env(ALLOW_RESTARTS) -m bool }
+    http-request deny unless METH_GET || METH_POST { env(POST) -m bool } || METH_DELETE { env(DELETE) -m bool }
```

Comment:
I don't see the `DELETE` env variable in the README or the Dockerfile.

Also if `POST=0 DELETE=0`, this will be denied now and won't continue evaluating any more rules, right? Then all new rules should exist before this one.